components/invitation_modal/invitation_modal_confirm_step.jsx

```diff
@@ -24,6 +24,43 @@ export default class InvitationModalConfirmStep extends React.Component {
         invitesNotSent: PropTypes.array.isRequired,
     }
 
+    getInvitesCountsMessage = (invitesSentCount, invitesNotSentCount) => {
+        if (invitesSentCount > 0 && invitesNotSentCount > 0) {
+            return (
+                <FormattedMarkdownMessage
+                    id='invitation_modal.confirm.members_subtitle'
+                    defaultMessage='**{sentCount, number} {sentCount, plural, one {person} other {people}}** {sentCount, plural, one {has} other {have}} been invited, and **{notSentCount, number} {notSentCount, plural, one {invitation} other {invitations}}** {notSentCount, plural, one {was} other {were}} not sent'
+                    values={{sentCount: invitesSentCount, notSentCount: invitesNotSentCount}}
+                />
+            );
+        }
+
+        if (invitesSentCount > 0 && invitesNotSentCount === 0) {
+            return (
+                <FormattedMarkdownMessage
+                    id='invitation_modal.confirm.members_subtitle_without_not_sent'
+                    defaultMessage='**{sentCount, number} {sentCount, plural, one {person} other {people}}** {sentCount, plural, one {has} other {have}} been invited'
+                    values={{sentCount: invitesSentCount}}
+                />
+            );
+        }
+        if (invitesSentCount === 0 && invitesNotSentCount > 0) {
+            return (
+                <FormattedMarkdownMessage
+                    id='invitation_modal.confirm.members_subtitle_without_sent'
+                    defaultMessage='**{notSentCount, number} {notSentCount, plural, one {invitation} other {invitations}}** {notSentCount, plural, one {was} other {were}} not sent'
+                    values={{notSentCount: invitesNotSentCount}}
+                />
+            );
+        }
+        return (
+            <FormattedMarkdownMessage
+                id='invitation_modal.confirm.members_subtitle_without_sent_and_not_sent'
+                defaultMessage='No invitation sent'
```

Comment:
@wiersgallak this string is new, maybe you want to decide to tune it a bit. Anyway this shouldn't happen, but just in case I added that code.

Reply:
@jespino would this happen because of an error?  Trying to determine what might trigger so I can better help with wording.  The header already has "Invitations not sent" so I'd like to be a little more explicit if possible with this message.

Reply:
It can't happen using the current interface, In theory it would happen if you don't add any user or email in the form and click "Invite" but for that cases, the button just doesn't work, so It can't happen.